ddtrace/internal/writer.py

```diff
@@ -298,7 +285,7 @@ def __init__(
                 }
             )
 
-        self._encoder = Encoder(
+        self._encoder = encoder_cls(
             max_size=self._buffer_size,
```

Comment:
We discussed moving these encoder parameters out of the writer in a separate refactor.